Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -248,6 +248,15 @@ extension PackageDescription.Package {
             dependencies = array.map { PackageDescription.Package.Dependency.fromJSON($0, baseURL: baseURL) }
         }
 
+        // Parse the compatible swift versions.
+        var compatibleSwiftVersions: [Int]? = nil
```

Comment:
Should we also check for an empty array here (and at least warn, if not give an error)?  My understanding is that if you provide an empty list, you won't ever be compatible with any tools, and that sounds like an unintentional thing worth at least warning about.